src/layer/tile/TileLayer.js

```diff
@@ -219,8 +227,8 @@ export var TileLayer = GridLayer.extend({
 		return zoom + zoomOffset;
 	},
 
-	_getSubdomain: function (tilePoint) {
-		var index = Math.abs(tilePoint.x + tilePoint.y) % this.options.subdomains.length;
+	_getSubdomain: function (tilePoint, failoverOffset = 0) {

```

Comment:
This "new" JS syntax is (currently) not supported.

Replace it with something like

```js
_getSubdomain: function (tilePoint, failoverOffset) {
  if(failoverOffset === undefined) failoverOffset = 0
```

I have #7649 pending to support this and other syntax features